examples/2_read_serial.cpp

```diff
@@ -60,8 +60,7 @@ int main()
 
     Offset chunk_offset = {1, 1, 1};
     Extent chunk_extent = {2, 2, 1};
-    std::unique_ptr< double[] > chunk_data;
-    E_x.loadChunk(chunk_offset, chunk_extent, chunk_data);
+    auto chunk_data = E_x.loadChunk<double>(chunk_offset, chunk_extent);
```

Comment:
I really wish the syntax `shared_ptr< auto > = someFunc< double >();` would exist...

Reply:
should I write `shared_ptr<double>` here so users know what is going on?

Reply:
As this is only the second piece of code a user would see, this is a good point in time to show them the way we handle raw data.